gulpfile.js

```diff
@@ -262,19 +246,17 @@ gulp.task('build:theme-fork:sass:generate-css', function () {
     .pipe(livereload())
 })
 
-gulp.task('build:theme-fork', function () {
-  gulp.start(
-    'build:theme-fork:sass:generate-css'
-  )
-})
+const buildThemeFork = gulp.parallel(
```

Comment:
Dit snap ik niet.

Reply:
Vroeger moest iedere task als `gulp.task()` gedefinieerd worden, nu is het niet meer nodig om een task te definiëren die enkel nog andere tasks uitvoert, vandaar een constante. `gulp.start()` bestaat ook niet meer, moet altijd `.series()` of `.parallel()` zijn.

Reply:
Ok